installation/versions.env

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
Please do not checkin the `versions.env` file.
This is automatically fetched/generated and should not be part of the source code.

Reply:
See above :)

Reply:
It is ok to have the versions file in this branch temporarily for dedicated cluster deployments, but we can't merge it into master.

So until it is removed, this PR is not accepted for merging.

Reply:
It is removed